<ui version="4.0" >
 <class>ExternalProgrammsDlg</class>
 <widget class="QWidget" name="ExternalProgrammsDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../simon.qrc" >:/images/icons/applications-other.svg</iconset>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="gbHTK" >
     <property name="title" >
      <string>HTK Programme</string>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QLabel" name="lbHDMan" >
          <property name="text" >
           <string>HDMan:</string>
          </property>
          <property name="buddy" >
           <cstring>leHDMan</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbHLEd" >
          <property name="text" >
           <string>HLEd:</string>
          </property>
          <property name="buddy" >
           <cstring>leHLEd</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbHCopy" >
          <property name="text" >
           <string>HCopy:</string>
          </property>
          <property name="buddy" >
           <cstring>leHCopy</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbHCompV" >
          <property name="text" >
           <string>HCompV:</string>
          </property>
          <property name="buddy" >
           <cstring>leHCompV</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QLineEdit" name="leHDMan" />
        </item>
        <item>
         <widget class="QLineEdit" name="leHLEd" />
        </item>
        <item>
         <widget class="QLineEdit" name="leHCopy" />
        </item>
        <item>
         <widget class="QLineEdit" name="leHCompV" />
        </item>
       </layout>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>150</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QLabel" name="lbHERest" >
          <property name="text" >
           <string>HERest:</string>
          </property>
          <property name="buddy" >
           <cstring>leHERest</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbHHEd" >
          <property name="text" >
           <string>HHEd:</string>
          </property>
          <property name="buddy" >
           <cstring>leHHEd</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbHVite" >
          <property name="text" >
           <string>HVite:</string>
          </property>
          <property name="buddy" >
           <cstring>leHVite</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QLineEdit" name="leHERest" />
        </item>
        <item>
         <widget class="QLineEdit" name="leHHEd" />
        </item>
        <item>
         <widget class="QLineEdit" name="leHVite" />
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbFileProcessing" >
     <property name="title" >
      <string>Dateibearbeitung</string>
     </property>
     <property name="flat" >
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLabel" name="lbBzip2" >
        <property name="text" >
         <string>Resample:</string>
        </property>
        <property name="buddy" >
         <cstring>leBzip2</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leBzip2" >
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../simon.qrc" />
 </resources>
 <connections/>
</ui>
